src/routes/+layout.svelte

```diff
@@ -105,5 +107,27 @@
 	</svelte:fragment>
 	<!-- Page Route Content -->
 	<slot />
-	<svelte:fragment slot="footer">Page Footer</svelte:fragment>
+	<svelte:fragment slot="pageFooter">
+			<div class="grid grid-cols-2 place-items-center md:!grid-cols-4 gap-1 lg:ml-10 lg:mr-10">
```

Comment:
```suggestion
			<div class="grid grid-cols-2 place-items-center md:!grid-cols-5 gap-1 lg:ml-10 lg:mr-10">
```
I'd add another col on md to give the description more room...